lang-java/gradle/autobuild/build

```diff
@@ -1,17 +1,25 @@
+abinfo "Creating version.txt ..."
+echo $PKGVER > "$SRCDIR"/version.txt
+
+abinfo "Initializing the output dir ..."
+install -dv "$SRCDIR"/temp
+
+abinfo "Building gradle ..."
+./gradlew installAll -Pgradle_installPath="$SRCDIR"/temp -PfinalRelease=true --no-configuration-cache --no-daemon
```

Comment:
Possibly bike-shedding, I prefer to explicitly specify the JDK version to use, or else it is possible for a wrong OpenJDK version (which is transitively included into BuildKit) to be used (and lead to incorrect class file versions).

Reply:
ah. didn't notice that.